### PR TITLE
clarify deprecation of `beacon_block` gossip topic in 4844

### PR DESCRIPTION
**Pull request by ralexstokes**
part of #3088

### CODE REVIEW

**Comment by terencechain** [comment]
Thank you! 🙏🏼

**Comment by jessepollak** [comment]
This looks good to me!

File: specs/eip4844/p2p-interface.md [hunk 1]
```diff
@@ -81,11 +82,17 @@ The new topics along with the type of the `data` field of a gossipsub message ar
 
 EIP4844 introduces a new global topic for beacon block and blobs-sidecars.
 
+##### `beacon_block`
+
+This topic is deprecated and clients **MUST NOT** expose in their topic set to any peer. Implementers do not need to do
```
Comment (djrtwo) [inline specs/eip4844/p2p-interface.md:87]: Might be worth noting when to turn it off. 
The two safe options are either:
1. (easier) Turn off the topic one epoch after `4844_FORK_EPOCH` to allow for any weird late gossip
2. (harder) turn off after `4844_FORK_EPOCH` is finalized
    * This is a bit more "correct" wrt to the `beacon_block` p2p slot range validation -- "validate that `signed_beacon_block.message.slot > compute_start_slot_at_epoch(state.finalized_checkpoint.epoch)`"
    * But might induce undue complexity when option (1) is pretty damn fine

File: specs/eip4844/p2p-interface.md [hunk 2]
```diff
@@ -81,11 +82,17 @@ The new topics along with the type of the `data` field of a gossipsub message ar
...
+anything beyond simply skip implementation and it is explicitly called out as it is a departure from previous versioning
```
Comment (djrtwo) [inline specs/eip4844/p2p-interface.md:88]:
```suggestion
anything beyond simply skip implementation, and it is explicitly called out as it is a departure from previous versioning
```

**Review by djrtwo — COMMENTED** [review]
Let's make it clear *when* to disable the topic

File: specs/eip4844/p2p-interface.md (hunk 1 above)
Comment (ralexstokes) [reply specs/eip4844/p2p-interface.md:87]: the spec here: 

https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/p2p-interface.md#transitioning-the-gossip

suggests to me that nodes should wait two epochs from the fork and then unsubscribe this topic

do you think we need special consideration here?

Comment (djrtwo) [reply specs/eip4844/p2p-interface.md:87]: This is fine. I might make an explicit call out to that previous area in the spec as I think it has the potential to be missed

**Comment by ralexstokes** [comment]
I think this is ready to go, just needs an approving review

**Review by djrtwo — APPROVED** [review]

**Pull request merged**